src/client/astro.config.mjs

```diff
@@ -1,12 +1,184 @@
-import { defineConfig } from 'astro/config';
+import { defineConfig } from "astro/config";
 import tailwind from "@astrojs/tailwind";
 import vue from "@astrojs/vue";
+import NetlifyCMS from "astro-netlify-cms";
 
 // https://astro.build/config
 export default defineConfig({
-  integrations: [vue(), tailwind({
-    config: {
-      applyBaseStyles: false
-    }
-  })]
-});
\ No newline at end of file
+  integrations: [
+    tailwind({
+      config: {
+        applyBaseStyles: false,
+      },
+    }),
+    NetlifyCMS({
+      config: {
+        backend: {
+          name: "test-repo",
```

Comment:
`netlify cms` is currently not configured with `GitHub OAuth`. That will come in my next PR.